extensions/amp-video-docking/validator-amp-video-docking.protoascii

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-video-docking
+  attr_lists: "common-extension-attrs"
+  extension_spec: {
+    name: "amp-video-docking"
+    requires_usage: NONE
```

Comment:
Why does this not require usage?

Reply:
The extension merely provisions a service, not a custom element.

Reply:
The service scans for elements with the `dock` attribute once it's registered. The service is used by merely including the extension, otherwise it would require creation of an `<amp-video-docking>` element, which this extension does not provide.

Reply:
As discussed offline, the semantics we need for `requires_usage` are based on attribute, not on custom element. The validator doesn't support this at the moment, so we can't require usage either way.